### PR TITLE
More descriptive icons for extract, attach, and image info

### PR DESCRIPTION
**Pull request by rdmark**
Icons drawn from scratch by yours truly in Inkscape, then cleaned up the source code manually.

### CODE REVIEW

File: python/web/src/static/themes/modern/icons/file-info.svg [hunk 1]
```diff
@@ -1 +1,5 @@
-<svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="feather feather-search"><circle cx="11" cy="11" r="8"></circle><line x1="21" y1="21" x2="16.65" y2="16.65"></line></svg>
\ No newline at end of file
+<svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="feather feather-link">
```
Comment (akuker) [inline python/web/src/static/themes/modern/icons/file-info.svg:1]: What about a circle with an "i" in it? In my mind, the question mark icon equates to "help?

Comment (akuker) [reply python/web/src/static/themes/modern/icons/file-info.svg:1]: ex: https://commons.wikimedia.org/wiki/File:Infobox_info_icon.svg

Comment (rdmark) [reply python/web/src/static/themes/modern/icons/file-info.svg:1]: Thanks for the feedback. I pushed a new icon. It used a blocky sans-serif type `i` with rounded edges to match the design language.

Comment (akuker) [reply python/web/src/static/themes/modern/icons/file-info.svg:1]: Looks awesome! Thank you!

**Review by akuker — APPROVED** [review]

**Pull request merged**